### DISCUSSION TITLE
Welcome to domino-game Discussions!

### DISCUSSION THREAD

**Discussion by eufelipemateus** [eufelipemateus/domino-game#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akkalame** [comment]
hey mate, good job with your domino game.
i have a question, why it no can play on both side of board?

**Reply by eufelipemateus** [reply]
You need 4 players to play this game.
